starport_template/lib/pages/select_asset_page.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:cosmos_ui_components/cosmos_theme.dart';
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+import 'package:starport_template/entities/balance.dart';
+import 'package:starport_template/pages/transfer_asset_page.dart';
+import 'package:starport_template/widgets/balance_card_list.dart';
+
+class SelectAssetPage extends StatelessWidget {
+  final List<Balance> balancesList;
+
+  const SelectAssetPage({Key? key, required this.balancesList}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: const CosmosAppBar(leading: CosmosBackButton(), title: 'Select Asset'),
+      body: Column(
+        children: [
+          SizedBox(height: CosmosTheme.of(context).spacingXXL),
+          BalanceCardList(
+            balancesList: balancesList,
+            onItemTapped: (balance) {
+              Navigator.of(context).push(MaterialPageRoute(builder: (context) => TransferAssetPage(balance: balance)));
+            },
+          ),
+        ],
```

Comment:
you could just wrap the `BalanceCardList` with padding. no need for the column then

Reply:
`BalanceCardList` is being used at one other place as well where the spacing from the top is different.